services/121-service/src/registration/enum/custom-data-attributes.ts

```diff
@@ -39,8 +39,9 @@ export enum GenericAttributes {
 }
 
 export class Attribute {
-  public attribute: string;
+  public name: string;
```

Comment:
Doesn't this change require changes in the PA-app and/or AW-app as well?

Or where these only ever used in the Portal?